lib/xclarity_client/services/update_policy_management.rb

```diff
@@ -0,0 +1,74 @@
+require 'json'
+
+module XClarityClient
+  class UpdatePolicyManagement < XClarityBase
+
+    include XClarityClient::ManagementMixin
+
+    def initialize(conf)
+      super(conf, UpdatePolicy::BASE_URI)
+    end
+
+    def population(opts = {})
+      get_all_resources(UpdatePolicy, opts)
+    end
+
+    def get_applicable_firmware()
+      get_update_policy("applicableFirmware", {})
+    end
+
+    def get_persisted_compare_results(opts = {})
+      get_update_policy("persistedResult", opts)
+    end
+    
+    def get_compare_results(opts = {})
+      get_update_policy("compareResult", opts)
+    end
+    
+    def get_update_policy(uri, opts = {})
+      if opts.size > 0
+         query = "?" + opts.map {|k, v| "#{k}=#{v}"}.join("&")
+      else
+         query = ""
+      end
+ 
+      response = connection(UpdatePolicy::BASE_URI + "/" + uri + query)
```

Comment:
Please, use the `@connection` defined on `Services::XClarityService` to do the requests for the LXCA. In this case you can do `@connection.do_get(managed_resource::BASE_URI)`

Reply:
thanks.. i have made respective update